api/services/events/v1/task.proto

```diff
@@ -5,15 +5,15 @@ package containerd.services.events.v1;
 option go_package = "github.com/containerd/containerd/api/services/events/v1;events";
 
 message TaskCreate {
-	string container_id = 1;
+	string id = 1;
```

Comment:
don't we want to also provide the `container-id` that task was created from? (same for Start and Delete)